### PR TITLE
Uses make create-demo-cluster

### PR DESCRIPTION
**Pull request by professor**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -138,7 +138,7 @@ source /usr/local/gpdb/greenplum_path.sh
 # Start demo cluster (gpdemo-env.sh is created which contain
 # __PGPORT__ and __MASTER_DATA_DIRECTORY__ values)
 cd gpAux/gpdemo
```
Comment (cjcjameson) [inline README.md]: Optional to remove the `cd`; if so, the `source` call afterwards will need a different path.

File: concourse/scripts/builds/GpBuild.py [hunk 2]
```diff
@@ -23,7 +23,7 @@ def icg(self):
             return status
         status = subprocess.call([
             "runuser gpadmin -c \"source /usr/local/gpdb/greenplum_path.sh \
-            && make cluster DEFAULT_QD_MAX_CONNECT=150\""], cwd="gpdb_src/gpAux/gpdemo", shell=True)
+            && make create-demo-cluster DEFAULT_QD_MAX_CONNECT=150\""], cwd="gpdb_src/gpAux/gpdemo", shell=True)
```
Comment (cjcjameson) [inline concourse/scripts/builds/GpBuild.py]: could use a different `cwd` now. Probably should genericize `gpdb_src` like https://github.com/greenplum-db/gpdb/pull/2479/files#diff-1431bece8773041fb410d099df26bac9R10

File: src/tools/docker/dotfiles/gpadmin.bashrc [hunk 3]
```diff
@@ -6,7 +6,7 @@ if [ ! -e /tmp/demo_sem ]; then
   touch /tmp/demo_sem
 
   pushd /workspace/gpdb/gpAux/gpdemo
```
Comment (cjcjameson) [inline src/tools/docker/dotfiles/gpadmin.bashrc]: Here we definitely don't need the `pushd` anymore

File: concourse/scripts/common.bash [hunk 4]
```diff
@@ -36,7 +36,7 @@ function make_cluster() {
   export DEFAULT_QD_MAX_CONNECT=150
   workaround_before_concourse_stops_stripping_suid_bits
   pushd gpdb_src/gpAux/gpdemo
```
Comment (cjcjameson) [inline concourse/scripts/common.bash]: Here we definitely don't need the `pushd` anymore

File: concourse/scripts/builds/GporcacodegenBuild.py [hunk 5]
```diff
@@ -25,7 +25,7 @@ def icg(self):
             return status
         status = subprocess.call([
             "runuser gpadmin -c \"source /usr/local/gpdb/greenplum_path.sh \
-            && make cluster\""], cwd="gpdb_src/gpAux/gpdemo", shell=True)
+            && make create-demo-cluster\""], cwd="gpdb_src/gpAux/gpdemo", shell=True)
```
Comment (cjcjameson) [inline concourse/scripts/builds/GporcacodegenBuild.py]: could use a different `cwd` now. Probably should genericize `gpdb_src` like https://github.com/greenplum-db/gpdb/pull/2479/files#diff-1431bece8773041fb410d099df26bac9R10

**Comment by professor** [comment]
@cjcjameson -- in looking at your proposed changes, I couldn't easily make then with confidence that the changes would work. If you want to work on these in the AM one morning, let me know.

**Pull request merged**